nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestUnpackContent.java

```diff
@@ -70,10 +70,14 @@ public void testTar() throws IOException {
         autoUnpackRunner.assertTransferCount(UnpackContent.REL_FAILURE, 0);
 
         final List<MockFlowFile> unpacked = unpackRunner.getFlowFilesForRelationship(UnpackContent.REL_SUCCESS);
+
         for (final MockFlowFile flowFile : unpacked) {
             final String filename = flowFile.getAttribute(CoreAttributes.FILENAME.key());
             final String folder = flowFile.getAttribute(CoreAttributes.PATH.key());
             final Path path = dataPath.resolve(folder).resolve(filename);
+            assertEquals(flowFile.getAttribute("file.inner.permission"),"420");
+            assertEquals("jmcarey", flowFile.getAttribute("file.inner.owner"));
+            assertEquals("mkpasswd", flowFile.getAttribute("file.inner.group"));
```

Comment:
Should have a test here for the last modified time attribute. You should be able to just use another `SimpleDateFormat` to see if you can successfully parse the attribute back into a `java.util.Date`.